tsl/test/t/004_truncate_or_delete_spin_lock.pl

```diff
@@ -97,10 +93,8 @@
 
 # Begin txns in both sessions
 $result = $s1->query_safe("BEGIN");
-isnt($result, '', "session 1: begin");
```

Comment:
why removing those `isnt`?? isn't it part of the necessary checks for this tap test?

Reply:
isnt empty string is not a meaningful test

Reply:
Nit: statement isn't a query, so we should probably use `safe_psql` here.